src/Parser.ts

```diff
@@ -122,13 +122,13 @@ export class Parser {
         word = st[i++];
 
         if (word === IRecordT.ERROR_INFORMATION) {
-          const version = st[i++];
-          fileId = st[i++];
+          const version = parseInt(st[i++]);
+          fileId = parseInt(st[i++]);
 
           let fileProcessed = this.sourceTable.get(fileId);
           if (!fileProcessed) {
-            if (fileId === ('000')) {
-              let location001 = this.sourceTable.get('001');
+            if (fileId === 0) {
```

Comment:
If file ID is '000' we can assume that it is the second file after the '999' output file